pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -548,6 +548,20 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 							operatorHealthImpactLabelKey: "critical",
 						},
 					},
+					{
+						Alert: "WindowsVirtualMachineMayReportOSDErrors",
```

Comment:
@machadovilaca  perhaps replace the name with
WindowsVirtualMachineOSDErrors, since its shorter and the runbook and description can explain what it means.

Also, Shouldn't we try to identify the message from the logs using Loki and only alert if the message exists?

Reply:
What about "WinVirtualMachineCRCErrors"? 

I'm not sure that the wording "When running windows VMs using default ODF storage with krbd, it **may report** bad crc/signature errors" is correct. My understanding is that there will be lots of CRC errors and cluster performance will be severely degraded. 

Are you planning to document a mitigation? There is a possibility that RXBounce might not be sufficient. Might require TXBounce as well. Things are getting complicated. See [BZ2190519#c44](https://bugzilla.redhat.com/show_bug.cgi?id=2190519#c44)

Reply:
updated the name and description. And yes, I will create the runbook afterward